Italian/main/SecurityCenter.apk/res/values-it/strings.xml

```diff
@@ -1456,87 +1456,87 @@ o inserisci la password"</string>
     <string name="apps_item_virus_summary_uninstalled">Apk</string>
     <string name="apps_item_sign_install_button_text">Sostituisci</string>
     <string name="apps_item_sign_dialog_title">"Installa versione ufficiale dell'app"</string>
-    <string name="apps_item_sign_dialog_content">"La firma di %s è diversa dalla versione presente nell\'App Store. Eliminare questa app ed installare la versione ufficiale? Nota: installare un'app con una firma diversa potrebbe causare la perdita di dati applicazione."</string>
+    <string name="apps_item_sign_dialog_content">"La firma di %s è diversa dalla versione presente nell'App Store. Eliminare questa app ed installare la versione ufficiale? Nota: installare un'app con una firma diversa potrebbe causare la perdita di dati applicazione."</string>
     <string name="wifi_item_title_connection">Connessione Wi-Fi</string>
     <string name="wifi_item_title_encryption">Crittografia Wi-Fi</string>
-    <string name="wifi_item_title_fake">Riconosci Wi-Fi fasulle</string>
+    <string name="wifi_item_title_fake">Individuazione Wi-Fi fasulle</string>
     <string name="wifi_item_title_dns">Sicurezza server DNS</string>
     <string name="wifi_item_title_arp_attack">Attacchi ARP</string>
     <string name="wifi_item_status_connection_yes">Connesso</string>
     <string name="wifi_item_status_connection_no">Non connesso</string>
     <string name="main_activity_content_title_scanning">Scansione…</string>
     <string name="main_activity_content_summary_root">Accesso root</string>
-    <string name="main_activity_content_summary_update">Aggiornamento di sistema</string>
+    <string name="main_activity_content_summary_update">Aggiornamento del sistema</string>
     <string name="dialog_title_stop_paysafety_scan">Interrompi</string>
     <string name="dialog_msg_stop_paysafety_scan">Interrompere il controllo di sicurezza per i pagamenti?</string>
-    <string name="result_wifi_header">Rilevati alcuni rischi Wi-Fi</string>
-    <string name="result_sms_header">Rilevati alcuni rischi SMS</string>
-    <string name="result_autoscan_header">Verifica pagamento</string>
-    <string name="result_autoscan_info">Verifica automaticamente il sistema quando vengono avviate app per pagamento</string>
-    <string name="result_virus_header">Trovati %d virus e app non sicure</string>
```

Comment:
La variabile numerica la eliminiamo solo se eliminata negli xml originali e quindi visualizzata erroneamente in UI come testo, altrimenti ci atteniamo alla stringa originale. Nonostante i problemi con il plurale è meglio ripristinarla. Stessa cosa per 1477.